### PR TITLE
Update building check strings

### PR DESCRIPTION
**Pull request by SilentSpike**
Previous detail string was not relevant to majority of the world and also not the most useful as it explained why the issue might exist rather than what the issue actually is and how to fix it.

Have instead added fix strings which explain how to resolve intersections for different circumstances. Think the title alone is pretty self explanatory for the intersections.

Have added strings to try and clarify the other issues (gap between buildings and small building).

### CODE REVIEW

File: analysers/analyser_osmosis_building_overlaps.py [hunk 1]
```diff
@@ -158,41 +158,37 @@ class Analyser_Osmosis_Building_Overlaps(Analyser_Osmosis):
...
-'''The intersection surface is probably due to inaccuracies in the
-cadastre/import tools.''')))
+            fix = fix)
         self.classs_change[2] = self.def_class(item = 0, level = 2, tags = ['building', 'geom', 'fix:chair'],
             title = T_('Large building intersection'),
-            detail = self.merge_doc(detail, T_(
-'''A large overlap. Requires a visual check (Bing, cadastre or
-survey).''')))
+            fix = self.merge_doc(fix, '\n\n', T_(
+'''Large intersections may also be a duplicated mapping - in which case delete the less accurate element.''')))
```
Comment (frodrigo) [inline analysers/analyser_osmosis_building_overlaps.py]:
```suggestion
'''Large intersections may also be a duplicated mapping - in which case delete the less accurate elements.''')))
```

File: analysers/analyser_osmosis_building_overlaps.py [hunk 2]
```diff
@@ -158,41 +158,37 @@ class Analyser_Osmosis_Building_Overlaps(Analyser_Osmosis):
...
-cadastre/import tools.''')))
+            detail = T_(
+'''It looks like these buildings should be physically joined, but they don't share nodes to indicate this.'''),
+            fix = T_('Connect the buildings by joining nodes where appropriate.'))
         self.classs_change[5] = self.def_class(item = 0, level = 1, tags = ['building', 'fix:chair'],
             title = T_('Large building intersection cluster'),
-            detail = self.merge_doc(detail, T_(
-'''Group of important overlaps. Major problem like a double import.''')))
+            fix = self.merge_doc(fix, '\n\n', T_(
+'''Large intersections may also be a duplicated mapping - in which case delete the less accurate element.''')))
```
Comment (frodrigo) [inline analysers/analyser_osmosis_building_overlaps.py]:
```suggestion
'''Large intersections may also be a duplicated mapping - in which case delete the less accurate elements.''')))
```

File: analysers/analyser_osmosis_building_overlaps.py [hunk 3]
```diff
@@ -158,41 +158,37 @@ class Analyser_Osmosis_Building_Overlaps(Analyser_Osmosis):
...
-'''Space separation is probably due to inaccuracies in the
-cadastre/import tools.''')))
+            detail = T_(
+'''It looks like these buildings should be physically joined, but they don't share nodes to indicate this.'''),
+            fix = T_('Connect the buildings by joining nodes where appropriate.'))
         self.classs_change[5] = self.def_class(item = 0, level = 1, tags = ['building', 'fix:chair'],
             title = T_('Large building intersection cluster'),
-            detail = self.merge_doc(detail, T_(
-'''Group of important overlaps. Major problem like a double import.''')))
+            fix = self.merge_doc(fix, '\n\n', T_(
```
Comment (frodrigo) [inline analysers/analyser_osmosis_building_overlaps.py]: I think `\n\n` are not required. `merge_doc` should already done the job.

Comment (SilentSpike) [reply analysers/analyser_osmosis_building_overlaps.py]: Ah okay, wasn't sure about that, thanks

File: analysers/analyser_osmosis_building_overlaps.py [hunk 4]
```diff
@@ -158,41 +158,37 @@ class Analyser_Osmosis_Building_Overlaps(Analyser_Osmosis):
...
-            detail = self.merge_doc(detail, T_(
-'''The intersection surface is probably due to inaccuracies in the
-cadastre/import tools.''')))
+            fix = fix)
         self.classs_change[2] = self.def_class(item = 0, level = 2, tags = ['building', 'geom', 'fix:chair'],
             title = T_('Large building intersection'),
-            detail = self.merge_doc(detail, T_(
-'''A large overlap. Requires a visual check (Bing, cadastre or
-survey).''')))
+            fix = self.merge_doc(fix, '\n\n', T_(
```
Comment (SilentSpike) [inline analysers/analyser_osmosis_building_overlaps.py]:
```suggestion
            fix = self.merge_doc(fix, T_(
```

**Comment by frodrigo** [comment]
Thank you.

**Pull request merged**